pkg/storage/system.go

```diff
@@ -103,7 +103,7 @@ const sqlListSystems = `
 	WHERE
 		status='LCID_ISSUED' AND
 		request_type='NEW' AND
-		(lcid='') IS NOT TRUE;
+		lcid <> '';
```

Comment:
This is pretty minor, but can we use `!=` instead of `<>` here? I had to go look up what `<>` did and I think it's pretty likely that devs will be more familiar with `!=`.

Reply:
From the schema, it looks like this is a nullable string with a constraint that ensures that any string is at least 6 characters long. Should this check be `IS NOT NULL` instead of comparing to an empty string?

Reply:
Good point! I'll update it.